dir.props

```diff
@@ -144,7 +144,7 @@
     <PackageThirdPartyNoticesFile>$(SourceDir).nuget/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
 
     <!-- This should be kept in sync with package details in src/.nuget/init/project.json -->
-    <RuntimeIdGraphDefinitionFile>$(PackagesDir)/Microsoft.NETCore.Platforms/1.0.1-rc2-23712/runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(PackagesDir)/Microsoft.NETCore.Platforms/1.0.2-beta-24224-02/runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
What we do is define the expected version in an msbuild property like such https://github.com/dotnet/corefx/blob/master/dir.props#L101 and then we run the validation task https://github.com/dotnet/corefx/blob/master/build.proj#L108 that will verify the versions we have defined are the versions referenced in all our project.json files.